### DISCUSSION TITLE
How to get Response Schema for any Github Entity for example Organization or Repository?

### DISCUSSION THREAD

**Discussion by aadeethyeah** [community/community#36893] [Apps, API and Webhooks]
For example for getting the repositories we call this api > /orgs/{org}/repos
But how to get only the Keys and not values. or How to get the Response Schema which is given on the REST API Doc

**Comment by LangLangBart** [comment]
### REST API
Ref: [How to extract all (also nested) key names with jq](https://stackoverflow.com/questions/46564720/how-to-extract-all-also-nested-key-names-with-jq/46565894#46565894) (4/Oct/17)

```zsh
gh api -X GET /orgs/cli/repos | jq '.[0] | paths | join(".")'
[
  "id",
  "node_id",
  "name",
  "full_name",
  "private",
  "owner",
  "owner.login",
  "owner.id",
  "owner.node_id",
  "owner.avatar_url",
  "owner.gravatar_id",
  "owner.url",
  "owner.html_url",
  "owner.followers_url",
  "owner.following_url",
  "owner.gists_url",
  "owner.starred_url",
  "owner.subscriptions_url",
  "owner.organizations_url",
  "owner.repos_url",
  "owner.events_url",
  "owner.received_events_url",
  "owner.type",
...
```

GitHub repo: [github/rest-api-description](https://github.com/github/rest-api-description)

---

### GraphQL API
[GitHub Docs - GraphQL is introspective](https://docs.github.com/en/graphql/guides/introduction-to-graphql#discovering-the-graphql-api)

Use the [Explorer](https://docs.github.com/en/graphql/overview/explorer) to experiment with GraphQL queries.

GitHub repo: [octokit/graphql-schema](https://github.com/octokit/graphql-schema)
